projects/1.16/assets/hanami/hanami/lang/zh_cn.json

```
@@ -1 +1,65 @@
-{}
\ No newline at end of file
+{
+    "_comment": "樱木类",
+    "block.hanami.stripped_sakura_log": "去皮樱花原木",
+    "block.hanami.sakura_log": "樱花原木",
+    "block.hanami.stripped_sakura_wood": "去皮樱花木",
+    "block.hanami.sakura_wood": "樱花木",
+    "block.hanami.sakura_planks": "樱花木板",
+    "block.hanami.sakura_slab": "樱花木台阶",
+    "block.hanami.sakura_stairs": "樱花木楼梯",
+    "block.hanami.sakura_pressure_plate": "樱花木压力板",
+    "block.hanami.sakura_fence": "樱花木栅栏",
+    "block.hanami.sakura_fence_gate": "樱花木栅栏门",
+    "block.hanami.sakura_button": "樱花木按钮",
+    "block.hanami.sakura_trapdoor": "樱花木活板门",
+    "block.hanami.sakura_door": "樱花木门",
+    "block.hanami.sakura_sign": "樱花木告示牌",
+    "block.hanami.sakura_leaves": "樱花树树叶",
+    "block.hanami.sakura_sapling": "樱花树树苗",
+    "block.hanami.sakura_leaf_carpet": "樱花叶地毯",
+    "block.hanami.sakura_bookshelf": "樱花木书架",
+    "block.hanami.sakura_chest": "樱花木箱子",
+    "block.hanami.sakura_trapped_chest": "樱花木陷阱箱",
+    "block.hanami.sakura_ladder": "樱花木梯子",
+    "block.hanami.sakura_vertical_slab": "樱花木竖直台阶",
+    "block.hanami.vertical_sakura_planks": "竖纹樱花木板",
+    "block.hanami.sakura_beehive": "樱花蜂箱",
+    "block.hanami.potted_sakura_sapling": "樱花树树苗盆栽",
+    "item.hanami.sakura_boat": "樱花木船",
+    "_comment2": "食物",
+    "effect.hanami.instability": "失稳",
+    "effect.hanami.nourishing": "滋润",
```

```suggestion
    "effect.hanami.nourishing": "滋补",
```
根据[wiki](https://docs.google.com/document/d/e/2PACX-1vSQQJnIRr4d1TBrtZJvMez7zShqHknLfYcqa7s9Wkfjn-F0egNbOG5zLN_uOjAz7RydxkNR1h911JnF/pub)，其效果为“带有状态时食用的食物会增加更多的饥饿值，推荐吃大量干海带前饮用（能提供此效果的 Sakura Cherry Juice）”

---

感觉可改可不改？

---

根据汉典，滋润为“用食物或营养品供应”，滋补为“滋养补益”，这个效果是饥饿值加成，理应滋养更佳。

---

```suggestion
    "effect.hanami.nourishing": "食物增强",
```